### PR TITLE
Increase coverage for extract scripts.

### PR DESCRIPTION
**Pull request by Leon-JS02**
Tests added for various utility functions in the Epic and Steam extract scripts.

### CODE REVIEW

File: web_scraping/steam_scraper/test_scrape_steam.py [hunk 1]
```diff
@@ -3,10 +3,102 @@
...
+    ('<div></div>', 'No description available.'),
+    ('', 'No description available.')
+])
+def test_scrape_game_description(html, expected):
+    """Tests that the scrape game description function can handle a range of inputs."""
+    soup = BeautifulSoup(html, 'html.parser')
+    assert scrape_game_description(soup) == expected
+
+
+@pytest.mark.parametrize("html, expected", [
```
Comment (MichaelHassan) [inline web_scraping/steam_scraper/test_scrape_steam.py:89]: nice testing more edge cases

**Review by MichaelHassan — APPROVED** [review]
I like that you are testing more of the code. This gives more confidence that it is robust.

File: web_scraping/steam_scraper/test_scrape_steam.py [hunk 2]
```diff
@@ -3,10 +3,102 @@
...
+    ('<div class="search_released">October 2020</div>', datetime(2020, 10, 1)),
+    ('<div class="search_released">13 Oct, 2020</div>', datetime(2020, 10, 13)),
+    ('<div class="search_released">2020-10-13</div>', datetime(2020, 10, 13)),
+    ('<div class="search_released"></div>', None),
+    ('<div></div>', None),
+    ('', None)
+])
+def test_parse_release_date(html, expected):
+    """Tests that the parse release date is capable of parsing a range of different date strings."""
+    soup = BeautifulSoup(html, 'html.parser')
```
Comment (mehrdadhalali) [inline web_scraping/steam_scraper/test_scrape_steam.py:100]: Does this not get patched?

**Review by mehrdadhalali — APPROVED** [review]

**Review by paulauus — APPROVED** [review]
Happy if tests are passing.

**Pull request merged**